docs/source/documentation/adrs/adr-011-use-separate-aws-accounts-for-data.html.md.erb

```diff
@@ -0,0 +1,60 @@
+---
+owner_slack: "#data-platform-notifications"
```

Comment:
Sneaky!

Reply:
We need a record of the why did you not....